### PR TITLE
OKTA-516607 : Widget vertical position height fix

### PR DESCRIPTION
**Pull request by leemchale-okta**
## Description:

This PR fixes the issue of the Sign in widget appearing too low on the page.

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [x] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-516607](https://oktainc.atlassian.net/browse/OKTA-516607)

### Screenshot/Video:
image

### CODE REVIEW

File: src/v3/src/components/AuthContainer/AuthContainer.tsx [hunk 1]
```diff
@@ -11,13 +11,11 @@
  */
 
 import { Box, useMediaQuery } from '@mui/material';
-import classNames from 'classnames';
 import { FunctionComponent, h } from 'preact';
 
 import style from './style.css';
 
 const AuthContainer: FunctionComponent = ({ children }) => {
-  const classes = classNames('auth-container', 'main-container', style.mainViewContainer);
```
Comment (glenfannin-okta) [inline src/v3/src/components/AuthContainer/AuthContainer.tsx:20]: I recall this was originally added to help preserve some of the classes that exists in v2 (i.e. if an org is hooking onto the `main-container` class name, to perform custom css changes, this was done to prevent breaking their customizations). Also, I can't recall if the existing v2 tests require one of these classes, if none of these are a concern any longer then I think it is ok to remove, otherwise, we may need to keep it.

Was removing these additional classes needed to make this work?

Comment (leemchale-okta) [reply src/v3/src/components/AuthContainer/AuthContainer.tsx:20]: Actually it was not necessary to remove them to make it work, it looked like the widget container was not inheriting any styles from those classes and my thinking was we were going to extract them anyway.  But on second thought I agree I think they still may be needed and I will add them back in.

**Review by glenfannin-okta — APPROVED** [review]

File: src/v3/src/components/Widget/index.tsx [hunk 2]
```diff
@@ -286,7 +286,8 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
     >
       {/* Note that we need two theme providers until we fully migrate to odyssey-mui */}
       <MuiThemeProvider theme={mapMuiThemeFromBrand(brandColors)}>
-        <ScopedCssBaseline>
+        {/* the style is to allow the widget to inherit the parent's bg color */}
+        <ScopedCssBaseline sx={{ backgroundColor: 'inherit' }}>
```
Comment (shawyu-okta) [inline src/v3/src/components/Widget/index.tsx:290]: just curious, this can't be done in the css of `AuthContainer`?

Comment (glenfannin-okta) [reply src/v3/src/components/Widget/index.tsx:290]: When I tried it on the auth container, the background color of the body was still not being inherited to the widget (I initially tried to set it there).

Comment (shawyu-okta) [reply src/v3/src/components/Widget/index.tsx:290]: Curious if this behavior is what we want -- Is there a discussion about this somewhere? Or an example screenshot of how this behaves?

Comment (glenfannin-okta) [reply src/v3/src/components/Widget/index.tsx:290]: So it was something I noticed when discussing the height issue with Lee. If you look at the current widget, they set the bg color in Monolith in the containing div 

image


If we mimic this with V3, you will notice the background color does not get applied because the MuiScopedCssBaseline div set the bg color to white. 
image

image

Not sure if the beta users will customize a background color or not but ultimately this is something I think we will need to support because if the widget is embedded in a company's website/page (custom hosted scenarios), the widget will set the containing bg color to white when it should inherit whatever div it is placed in). IIRC, we should not control colors/styling outside of the container, whereas this MuiScopedCssBaseline div is by setting the bg to white.

Comment (leemchale-okta) [reply src/v3/src/components/Widget/index.tsx:290]: I believe this is in parity with V2 @glenfannin-okta is that correct?

Comment (shawyu-okta) [reply src/v3/src/components/Widget/index.tsx:290]: Oh I see, thank you for the explanation - I was thinking this was affecting the inner container of the widget. Agreed that the larger surrounding container should not set its own bgcolor and instead should inherit the surrounding container

**Review by shawyu-okta — APPROVED** [review]
